.gitignore

```diff
@@ -1,7 +1,11 @@
-.idea
+**/.idea
+**/.vscode
+**/node_modules
+**/*.iml
+**/.directory
+**/logs
+**/dist
+**/config/config.toml
 .run
 mqtt5
-**/.yarn
-*/.pnp.cjs
-*/.pnp.loader.mjs
```

Comment:
ca fais pas sens de les garder just in case ?

Reply:
disons que non tfacon ca passera jamais une review ca

Reply:
ça évite que les gens utilise yarn, au moins ils verront si y a les fichiers ajoutés à git